### DISCUSSION TITLE
Would like to make a PR but the Github tests fail...

### DISCUSSION THREAD

**Discussion by superDuperCyberTechno** [laravel/framework#50529] [Q&A]
Is there a guide where I can learn how to accomodate the Github testing? I've looked at the official page on the subject (https://laravel.com/docs/11.x/contributions) but that doesn't seem to cover anything.

I've made 2 pull requests that both failed and I really have no idea how to handle them.

https://github.com/laravel/framework/pull/50515
https://github.com/laravel/framework/pull/50528

Am I missing something really basic here? 

**It may be worth noting that I've made all code edits on Github itself, the repo has never touched my development machine.**

**Comment by iazaran** [comment]
If you want to check the issues you can click on failed test icons:
![image](https://github.com/laravel/framework/assets/10566709/61a847e3-3c56-470b-ad9a-9bcfe6cccea9)
And then go to the failed details.
